src/components/pill/index.js

```diff
@@ -0,0 +1,123 @@
+import { html, css, LitElement, unsafeCSS } from 'lit';
+import styles from './styles.scss';
+
+const props = {
+  disabled: 'data-vl-disabled',
+  type: 'data-vl-type',
+  closable: 'data-vl-closable',
+  checkable: 'data-vl-checkable',
+};
+
+const { disabled, type, closable, checkable } = props;
+
+export class VlPill extends LitElement {
+  static get EVENTS() {
+    return {
+      close: 'close',
+      check: 'check',
+    };
+  }
+
+  static get styles() {
+    return [
+      css`
+        ${unsafeCSS(styles)}
+      `,
+    ];
+  }
+
+  static get properties() {
+    return {
+      [disabled]: { type: Boolean },
```

Comment:
Nu gebruik je properties die "data-vl-...' noemen. Je property kan gewoon bv 'disabled' noemen, het is best practice om wijzigingen van die property te reflecteren in het attribuut en andersom. Lit heeft dat ingebakken.


Voorbeeld accessibility statement: https://github.com/milieuinfo/uig-webcomponents/blob/master/src/components/accessibility/index.js

Reply:
Zou moeten ok zijn nu.